python/feldera/_callback_runner.py

```diff
@@ -0,0 +1,64 @@
+from threading import Thread
+from typing import Callable
+from queue import Queue, Empty
+
+import pandas as pd
+from feldera import FelderaClient
+from feldera._helpers import dataframe_from_response
+from feldera.output_handler import _OutputHandlerInstruction
+
+
+class CallbackRunner(Thread):
+    def __init__(
+            self,
+            client: FelderaClient,
+            pipeline_name: str,
+            view_name: str,
+            callback: Callable[[pd.DataFrame, int], None],
+            queue: Queue,
+    ):
```

Comment:
This code is almost identical to `output_handler.rs`.  I understand that that code assembles the result in a single dataframe, while here we invoke the callback for each batch.  But I would expect the former to be implemented on top of the latter, so we shouldn't need two near-identical implementations.

Reply:
Yes. I am working on finding an elegant way to merge the two.